ERROR | 2025-03-24 22:38:38 | Expected status 'closed', but got 'performing' for route e6d40f50-785f-4fe4-af29-05fcc9df3e8a. | file: tests/e2e/test_delivery_couier_flow.py::TestDeliveryCourierFlow::test_multiple_deliveries_routing_flow (setup)
ERROR | 2025-03-24 22:38:44 | Expected status 'closed', but got 'performing' for route e6d40f50-785f-4fe4-af29-05fcc9df3e8a. | file: tests/e2e/test_delivery_couier_flow.py::TestDeliveryCourierFlow::test_multiple_deliveries_routing_flow (setup)
ERROR | 2025-03-24 22:38:44 | Expected status 'closed', but got 'performing' for route e6d40f50-785f-4fe4-af29-05fcc9df3e8a. | file: tests/e2e/test_delivery_couier_flow.py::TestDeliveryCourierFlow::test_multiple_deliveries_routing_flow (setup)
ERROR | 2025-03-24 22:38:49 | Expected status 'closed', but got 'performing' for route e6d40f50-785f-4fe4-af29-05fcc9df3e8a. | file: tests/e2e/test_delivery_couier_flow.py::TestDeliveryCourierFlow::test_multiple_deliveries_routing_flow (setup)
ERROR | 2025-03-24 22:38:49 | Expected status 'closed', but got 'performing' for route e6d40f50-785f-4fe4-af29-05fcc9df3e8a. | file: tests/e2e/test_delivery_couier_flow.py::TestDeliveryCourierFlow::test_multiple_deliveries_routing_flow (setup)
ERROR | 2025-03-24 22:38:49 | Expected status 'closed', but got 'performing' for route e6d40f50-785f-4fe4-af29-05fcc9df3e8a. | file: tests/e2e/test_delivery_couier_flow.py::TestDeliveryCourierFlow::test_multiple_deliveries_routing_flow (setup)
ERROR | 2025-03-24 22:38:56 | Expected 400 status code but got 500 status code instead | file: tests/test_company/test_get_companies.py::TestGetCompanies::test_pagination[0-10-HTTPStatus.BAD_REQUEST] (setup)
ERROR | 2025-03-24 22:38:56 | Expected 400 status code but got 500 status code instead | file: tests/test_company/test_get_companies.py::TestGetCompanies::test_pagination[-1-10-HTTPStatus.BAD_REQUEST] (setup)
ERROR | 2025-03-24 22:39:05 | Expected 400 status code but got 301 status code instead | file: tests/test_company/test_get_company_by_id.py::TestGetCompanyById::test_get_company_by_invalid_id[] (setup)
ERROR | 2025-04-11 18:42:46 | Expected 201 status code but got 400 status code instead | file: tests/test_company/test_create_company.py::TestCreateCompany::test_create_saas_company_with_invoice_customer_ids (setup)
ERROR | 2025-04-11 18:48:55 | 9 validation errors for CompanyDetailsSchema
id
  Field required [type=missing, input_value={'message': 'it is forbid...ent integration source'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
name
  Field required [type=missing, input_value={'message': 'it is forbid...ent integration source'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
is_external
  Field required [type=missing, input_value={'message': 'it is forbid...ent integration source'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
integration_source
  Field required [type=missing, input_value={'message': 'it is forbid...ent integration source'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
auto_assign_enabled
  Field required [type=missing, input_value={'message': 'it is forbid...ent integration source'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
eta_enabled
  Field required [type=missing, input_value={'message': 'it is forbid...ent integration source'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
external_license
  Field required [type=missing, input_value={'message': 'it is forbid...ent integration source'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
check_courier_slot_enabled
  Field required [type=missing, input_value={'message': 'it is forbid...ent integration source'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
created_at
  Field required [type=missing, input_value={'message': 'it is forbid...ent integration source'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing | file: src.validator
ERROR | 2025-04-11 18:54:47 | Expected 201 status code but got 400 status code instead | file: tests/test_company/test_create_company.py::TestCreateCompany::test_create_iiko_company_with_invoice_customer_ids (setup)
ERROR | 2025-04-11 19:31:53 | Expected status 'closed', but got 'performing' for route 1341c23d-8da7-455b-bf48-d640c7814578. | file: tests/e2e/test_delivery_couier_flow.py::TestDeliveryCourierFlow::test_multiple_deliveries_routing_flow (setup)
ERROR | 2025-04-11 19:31:58 | Expected status 'closed', but got 'performing' for route 1341c23d-8da7-455b-bf48-d640c7814578. | file: tests/e2e/test_delivery_couier_flow.py::TestDeliveryCourierFlow::test_multiple_deliveries_routing_flow (setup)
ERROR | 2025-04-11 19:31:58 | Expected status 'closed', but got 'performing' for route 1341c23d-8da7-455b-bf48-d640c7814578. | file: tests/e2e/test_delivery_couier_flow.py::TestDeliveryCourierFlow::test_multiple_deliveries_routing_flow (setup)
ERROR | 2025-04-11 19:32:03 | Expected status 'closed', but got 'performing' for route 1341c23d-8da7-455b-bf48-d640c7814578. | file: tests/e2e/test_delivery_couier_flow.py::TestDeliveryCourierFlow::test_multiple_deliveries_routing_flow (setup)
ERROR | 2025-04-11 19:32:03 | Expected status 'closed', but got 'performing' for route 1341c23d-8da7-455b-bf48-d640c7814578. | file: tests/e2e/test_delivery_couier_flow.py::TestDeliveryCourierFlow::test_multiple_deliveries_routing_flow (setup)
ERROR | 2025-04-11 19:32:03 | Expected status 'closed', but got 'performing' for route 1341c23d-8da7-455b-bf48-d640c7814578. | file: tests/e2e/test_delivery_couier_flow.py::TestDeliveryCourierFlow::test_multiple_deliveries_routing_flow (setup)
ERROR | 2025-04-14 11:51:33 | Failed to fetch courier ID: 400. URL: /user. | file: services.auth_service
ERROR | 2025-04-14 11:54:12 | Failed to fetch courier ID: 400. URL: https://logistics.bsl.dev/api/user. | file: services.auth_service
ERROR | 2025-04-15 20:41:05 | Expected 400 status code but got 500 status code instead | file: tests/test_company/test_get_companies.py::TestGetCompanies::test_pagination[0-10-HTTPStatus.BAD_REQUEST] (setup)
ERROR | 2025-04-15 20:41:06 | Expected 400 status code but got 500 status code instead | file: tests/test_company/test_get_companies.py::TestGetCompanies::test_pagination[-1-10-HTTPStatus.BAD_REQUEST] (setup)
ERROR | 2025-04-15 20:41:08 | Expected 400 status code but got 301 status code instead | file: tests/test_company/test_get_company_by_id.py::TestGetCompanyById::test_get_company_by_invalid_id[] (setup)
ERROR | 2025-04-15 22:02:27 | Authentication failed: 401 for role logistician_iiko | file: services.auth_service
ERROR | 2025-04-15 22:04:24 | Authentication failed: 401 for role logistician_iiko | file: services.auth_service
ERROR | 2025-04-15 22:06:18 | Authentication failed: 401 for role logistician_iiko | file: services.auth_service
ERROR | 2025-04-15 22:07:29 | Authentication failed: 401 for role logistician_iiko | file: services.auth_service
ERROR | 2025-04-28 13:06:44 | 56 validation errors for DeliveriesListSchema
deliveries.0.events.0.author
  Input should be an object [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/model_type
deliveries.0.time_to_assigns
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.0.last_minute
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.0.route_step_points_fact
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.0.route_step_points_plan
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.1.events.0.author
  Input should be an object [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/model_type
deliveries.1.time_to_assigns
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.1.last_minute
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.1.route_step_points_fact
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.1.route_step_points_plan
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.2.events.0.author
  Input should be an object [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/model_type
deliveries.2.time_to_assigns
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.2.last_minute
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.2.route_step_points_fact
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.2.route_step_points_plan
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.3.events.1.author
  Input should be an object [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/model_type
deliveries.3.time_to_assigns
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.3.last_minute
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.3.route_step_points_fact
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.3.route_step_points_plan
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.4.courier
  Input should be a valid string [type=string_type, input_value={'user_id': 'aaefc2fb-57f...up_points_online': None}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/string_type
deliveries.4.events.5.author
  Input should be an object [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/model_type
deliveries.4.time_to_assigns
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.4.last_minute
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.4.route_step_points_fact
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.4.route_step_points_plan
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.5.courier
  Input should be a valid string [type=string_type, input_value={'user_id': 'aaefc2fb-57f...up_points_online': None}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/string_type
deliveries.5.events.5.author
  Input should be an object [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/model_type
deliveries.5.time_to_assigns
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.5.last_minute
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.5.route_step_points_fact
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.5.route_step_points_plan
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.6.courier
  Input should be a valid string [type=string_type, input_value={'user_id': 'aaefc2fb-57f...up_points_online': None}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/string_type
deliveries.6.events.5.author
  Input should be an object [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/model_type
deliveries.6.time_to_assigns
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.6.last_minute
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.6.route_step_points_fact
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.6.route_step_points_plan
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.7.courier
  Input should be a valid string [type=string_type, input_value={'user_id': 'aaefc2fb-57f...up_points_online': None}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/string_type
deliveries.7.events.5.author
  Input should be an object [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/model_type
deliveries.7.time_to_assigns
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.7.last_minute
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.7.route_step_points_fact
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.7.route_step_points_plan
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.8.courier
  Input should be a valid string [type=string_type, input_value={'user_id': 'aaefc2fb-57f...up_points_online': None}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/string_type
deliveries.8.events.5.author
  Input should be an object [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/model_type
deliveries.8.time_to_assigns
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.8.last_minute
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.8.route_step_points_fact
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.8.route_step_points_plan
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.9.courier
  Input should be a valid string [type=string_type, input_value={'user_id': 'aaefc2fb-57f...up_points_online': None}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/string_type
deliveries.9.events.5.author
  Input should be an object [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/model_type
deliveries.9.time_to_assigns
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.9.last_minute
  Input should be an object [type=dict_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/dict_type
deliveries.9.route_step_points_fact
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type
deliveries.9.route_step_points_plan
  Input should be a valid array [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/list_type | file: src.validator
ERROR | 2025-04-28 13:21:19 | 2 validation errors for DeliveryDetailsSchema
specification.delivered_partially
  Field required [type=missing, input_value={'items': [{'id': '08e3b2...ers': [], 'items': []}]}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
events.0.author
  Input should be an object [type=model_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/model_type | file: src.validator
ERROR | 2025-04-28 13:27:52 | 1 validation error for DeliveryDetailsSchema
specification.delivered_partially
  Field required [type=missing, input_value={'items': [{'id': '7b3525...ers': [], 'items': []}]}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing | file: src.validator
